9/changelog.yaml

```
@@ -2,6 +2,13 @@ ancestor: 8.0.0
 releases:
   9.0.0a1:
     changes:
+      minor_changes:
```

I wonder whether this should be moved to
```suggestion
      major_changes:
```
so it shows up in the porting guide? On the other hand, this should not really affect anyone directly. WDYT?

---

Yeah, it shouldn't really affect anyone except maybe Linux distributions that patch the setup.py in [some way](https://src.fedoraproject.org/rpms/ansible/blob/0321735d7fc291639db0b22572e0d5bd812b7d97/f/ansible.spec#_91). I'm fine with adding it to the Porting Guide, but I don't think it *needs* to be there.

---

In that case, let's keep it as-is.